code/src/java/pcgen/facade/core/CharacterFacade.java

```diff
@@ -71,71 +71,71 @@
 public interface CharacterFacade extends CompanionFacade
 {
 
-	public InfoFactory getInfoFactory();
+	InfoFactory getInfoFactory();
 
-	public ReferenceFacade<Gender> getGenderRef();
+	ReferenceFacade<Gender> getGenderRef();
 
-	public void setAlignment(PCAlignment alignment);
+	void setAlignment(PCAlignment alignment);
 
-	public ReferenceFacade<PCAlignment> getAlignmentRef();
+	ReferenceFacade<PCAlignment> getAlignmentRef();
 
-	public void setGender(Gender gender);
+	void setGender(Gender gender);
 
-	public void setGender(String gender);
+	void setGender(String gender);
 
 	/**
 	 * @param stat The stat to retrieve the base for
 	 * @return A reference to the base score for the stat
 	 */
-	public ReferenceFacade<Number> getScoreBaseRef(PCStat stat);
+    ReferenceFacade<Number> getScoreBaseRef(PCStat stat);
```

Comment:
looks like whitespace is out

Reply:
Strange, CheckStyle hasn’t complained about it. I will correct this. 
By the way, publics were removed, because they are redundant in the interface.

Reply:
I don't know how this happened, but this line used spaces instead of tabs.